src/scenes/ProgressReports/Detail/ProgressReportDrawer/ProgressReportStepper.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Box,
+  Paper,
+  Step,
+  StepButton,
+  StepConnector,
+  Stepper,
+  Theme,
+  Typography,
+} from '@mui/material';
+import { PeopleJoinedIcon } from '~/components/Icons';
+import { useProgressReportContext } from '../../ProgressReportContext';
+import { colorPalette } from './colorPalette';
+
+interface ProgressReportStepperProps {
+  step: number;
+}
+
+const stepperSx = (theme: Theme) => ({
+  '& .MuiStep-root .MuiStepLabel-label': {
+    fontSize: '0.75rem',
+  },
+  '& .MuiStep-root': {
+    cursor: 'pointer',
+  },
+  '& .MuiStep-root .MuiStepIcon-root ': {
+    color: theme.palette.grey[500],
+  },
+  '& .MuiStep-root .MuiStepIcon-root.Mui-active': {
+    color: theme.palette.info.light,
+  },
+  '& .MuiStep-root .MuiStepIcon-root.Mui-completed': {
+    color: theme.palette.grey[500],
+    backgroundColor: theme.palette.grey[500],
+    borderRadius: '50%',
+  },
+  '& .MuiStepConnector-lineVertical': {
+    minHeight: 16,
+  },
+  '& .MuiStepLabel-root': {
+    paddingTop: 0.5,
+    paddingBottom: 0.5,
+  },
+});
+
+const typographySx = {
```

Comment:
Also these could be typed with `Sx`.
```tsx
const typographySx: Sx = { ... };
```
That would also allow us to avoid referencing the theme type and be more consistent with above
```tsx
const stepperSx: Sx = (theme) => ({
```